MINT.postman_collection.json

```diff
@@ -2256,7 +2256,7 @@
 						"body": {
 							"mode": "graphql",
 							"graphql": {
-								"query": "query currentUser {\n  currentUser {\n    notificationPreferences {\n        dailyDigestComplete\n\n        addedAsCollaborator\n\n        taggedInDiscussion\n\n        taggedInDiscussionReply\n\n        newDiscussionReply\n\n        modelPlanShared\n\n        newModelPlan\n\n        datesChanged\n        datesChangedNotificationType\n      }\n    }\n  }\n\n",
+								"query": "query currentUser {\n  currentUser {\n    notificationPreferences {\n        dailyDigestComplete\n\n        addedAsCollaborator\n\n        taggedInDiscussion\n\n        taggedInDiscussionReply\n\n        newDiscussionReply\n\n        modelPlanShared\n\n        newModelPlan\n\n        datesChanged\n        datesChangedNotificationType\n\n        dataExchangeApproachMarkedComplete\n      }\n    }\n  }\n\n",
```

Comment:
In order to move this along faster, I simply exported the integrated Post Man collection from the notification PR. I don't see a purpose to disintegrate this and reintegrate it since it's all going to a feature PR, not main.